packages/spectral/src/types/IntegrationDefinition.ts

```diff
@@ -50,6 +50,21 @@ export interface IntegrationDefinitionComponentRegistry {}
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface IntegrationDefinitionConfigPages {}
 
+/**
+ * Root UserLevelConfigPages type exposed for augmentation.
+ *
+ * The expected interface when augmenting is:
+ *
+ * ```ts
+ * interface IntegrationDefinitionUserLevelConfigPages {
+ *   [key: string]: ConfigPage
+ * }
+ * ```
+ *
+ */
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
```

Comment:
Wonder at this point if we just turn this rule off in the eslint rules `.eslintrc.json`

Reply:
Nope. Pretty explicit that we only want to allow it here.